Jenkinsfile

```diff
@@ -31,11 +31,11 @@ pipeline {
           },
           jenkins_master: {
             dir('ansible-jenkins') {
-              git branch: 'master', url: 'https://github.com/liatrio/ansible-jenkins.git'
+              git branch: 'PA-21-deployments', url: 'https://github.com/liatrio/ansible-jenkins.git'
```

Comment:
Is this supposed to be master?

Reply:
Correct! Good catch, new commit incoming.